src/conf/local-file.yaml

```
@@ -131,3 +131,12 @@ pprof: false
 # Discovered Policies Configuration  
 dsp:
   auto-deploy-dsp: false
+
+# Feeder Service 
+feederservice:
+  enable: true
+  tls:
+    cert:
+      file: /home/alok-pc/go/src/github.com/aloksharma20/discovery-engine/src/cert/ca-cert.pem
+    key:
+      file: /home/alok-pc/go/src/github.com/aloksharma20/discovery-engine/src/cert/ca-key.pem
```

```suggestion
server:
  port: 9089
  tls:
    enable: false
    cert: /tls/cert.pem
    key: /tls/key.pem
```
- Let's use this structure for the configuration.
- Currently we have hardcoded the server port in `grpcServer.go`. Let's make it configurable.